application/src/main/resources/application.yml

```diff
@@ -77,8 +77,8 @@ app:
     gmcUsername: ${GMC_USER_NAME:guest}
     gmcPassword: ${GMC_PASSWORD:guest}
     designatedBodies: ${DESIGNATED_BODY_CODE:1-AIIDHJ,1-AIIDMQ,1-AIIDNQ,1-AIIDMY,1-AIIDQQ,1-AIIDWT,1-AIIDR8,1-AIIDSA,1-AIIDH1,1-AIIDWA,1-AIIDVS,1-AIIDWI,1-AIIDSI}
-    recommendationstatuscheck.cronExpression: ${REC_STATUS_CHECK_CRON:0 0 * * * *}
-    nightlySyncStart.cronExpression: ${REC_NIGHTLY_SYNC_CRON:0 0 * * * *}
+    recommendationstatuscheck.cronExpression: ${REC_STATUS_CHECK_CRON:0 0 0 * * *}
+    nightlySyncStart.cronExpression: ${REC_NIGHTLY_SYNC_CRON:0 0 0 * * *}
```

Comment:
I am not sure these are right.  Did you want:
a) hourly at the start of the hour `0 * * * *`, and
b) daily at midnight `0 0 * * *`?